department-of-veterans-affairs/caseflow
# Special issues copy changes and new component

Resolves #7216

### Description
Update special issues copy in attorney/judge checkout flow per new mockups.
Since special issues are re-used in different applications, created a new file called `QUEUE_SPECIAL_ISSUES` and added in keys related to the section they should be in and the order. Used the existing `CheckboxGroup` component for the sections of checkboxes.
followed existing pattern of setting snakecase value of special issue in to `special_issues` node in redux store and setting to true/false if user has checked/unchecked that option.

new special issues: `committee_on_waivers_and_compromises` and `dic_united_states`

these are being sent in the post body, but are not being returned by the special issues api (`/appeals/${appeal.externalId}/special_issues`) on a 200 repsonse which means the backend needs to be updated to handle these new special issues? @aroltsch @lowellrex 

### Acceptance Criteria

### Testing Plan
1. Login as BVASCASPER1
2. Go to their queue /queue
3. Click on a legacy appeal
4. Under available actions, select ready for decision option
5. check the redux store in redux dev tools or check the console output to see that special issues are persisted in redux when toggling the checkboxes
6. when hitting continue, ensure that the special issues api returns a 200
